src/pybind/mgr/dashboard/frontend/src/app/ceph/pool/pool-form/pool-form.component.html

```diff
@@ -356,6 +356,20 @@
                   <ul ngbNav
                       #crushInfoTabs="ngbNav"
                       class="nav-tabs">
+                    <li ngbNavItem="crush-rule-description">
+                      <a ngbNavLink
+                         i18n>Description</a>
+                      <ng-template ngbNavContent>
+                        <span i18n>CRUSH Rules define policy about how data is
+                                   distributed across the devices in the hierarchy.They define placement and
+                                   replication strategies or distribution policies that allow you to specify
+                                   exactly how CRUSH places data replicas. For a more detailed description,
+                          refer to
+                          <a href="https://docs.ceph.com/en/latest/rados/operations/crush-map/#rules">
+                            CRUSH Rules Docs</a>
+                        </span>
+                      </ng-template>
```

Comment:
IMHO if we want* to add this piece of information we should use the `<cd-helper>` we are using for quota fields.

\* I find a bit confusing the text here: we should agree on what kind of user is using Ceph Dashboard. If we assume users are skilled on Ceph (that's my assumption given the amount of advanced Ceph settings and information we display), the only tooltips we should provide are to help them understand how Dashboard handles the input fields (like in the **Quota max objects**: _"Leave it blank or specify 0 to disable this quota"_), but not embedding an interactive Coursera training on Ceph basics.

Reply:
Last thing to mention: external links are _hardcoded dependencies_ to elements that we have limited control over:
* This link points to master so when master becomes Pacific we might need to update that.
* Additionally links can change due to documentation refactors.
* For downstream adaptations we'll need to change this again (`DocService` would be the place)

In case we want to go on adding more external links, I'd suggest to create a cross-link validation stage (similar to what @bk201 did for Grafana URLs) between dashboard and `/doc` directory.